src/setupWorker/glossary.ts

```diff
@@ -37,6 +37,11 @@ export type ServiceWorkerInstanceTuple = [
   ServiceWorkerRegistration,
 ]
 
+export type ServiceWorkerMatcher = (
+  scriptUrl: string,
+  absoluteWorkerUrl: string,
```

Comment:
It's a little confusing to have two arguments different in URL relativity to be called differently: script and worker.
I vote for naming them both `script`, as that is the official name of the worker script.

```diff
scriptUrl: string,
-absoluteWorkerUrl: string,
+absoluteScriptUrl: string
```